tljh/traefik-dynamic.toml.tpl

```diff
@@ -0,0 +1,25 @@
+# traefik.toml dynamic config (mostly TLS)
+# dynamic config in the static config file will be ignored
+{% if https['enabled'] %}
+[tls]
+  [tls.options.default]
+  minVersion = "VersionTLS12"
+
```

Comment:
```suggestion
  minVersion = "VersionTLS12"
  cipherSuites = [
    "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
    "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
    "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
    "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
    "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
    "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305"
    ]
```

while this is being fixed we should probably also sync this up with [z2jh config](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/73d4f182cc75d3761af1210ac53e81a17e400fe3/jupyterhub/templates/proxy/autohttps/_configmap-dynamic.yaml#LL101C1-L106C47). This also resolves https://github.com/jupyterhub/the-littlest-jupyterhub/issues/513